arch/arm/src/nrf52/nrf52_sdc.h

```diff
@@ -0,0 +1,65 @@
+/****************************************************************************
+ * /home/v01d/coding/nuttx_sdc/nuttx/arch/arm/src/chip/nrf52_sdc.h
```

Comment:
Please use relative path instead absolute path:
arch/arm/src/chip/nrf52_sdc.h

Reply:
Right, thanks. I need to fix my code template.

Reply:
done